connect/cli/core/utils.py

```diff
@@ -63,3 +64,26 @@ def validate_output_options(output_path, output_file, default_dir_name, default_
     output_file = os.path.join(output_path, output_file or f'{default_file_name}.xlsx')
 
     return output_file
+
+
+def field_to_check_mark(predicate, false_value=''):
```

Comment:
It is nice to have comments for public interfaces for that they are and what are the parameters